en/config/safety.md

```diff
@@ -0,0 +1,235 @@
+* GF_COUNT (INT32)	- Geofence counter limit. Comment: Set how many subsequent position measurements outside of the fence are needed before geofence violation is triggered
+* GF_SOURCE (INT32)	- Geofence source
+-->
+
+
+### Return Home Settings {#return_settings}
+
+*Return to land* is a common [failsafe action](#failsafe_actions) that engages [Return mode](../flight_modes/return.md) to return the vehicle to the home position. 
+This section shows how to set the land/loiter behaviour after returning.
+
+![Safety - Return Home Settings (QGC)](../../images/qgc/setup/safety_return_home.png)
+
+The settings and underlying parameters are shown below:
+
+Setting | Parameter | Description
+--- | --- | ---
+Climb to altitude | [RTL_RETURN_ALT](../advanced_config/parameter_reference.md#RTL_RETURN_ALT) | Vehicle ascend to this minimum height (if below it) for the return flight.
+Return behaviour |  | Choice list of *Return then*: Land, Loiter and do not land, or Loiter and land after a specified time.
+Loiter Altitude | [RTL_DESCEND_ALT](../advanced_config/parameter_reference.md#RTL_DESCEND_ALT) | If return with loiter is selected you can also specify the altitude at which the vehicle hold.
+Loiter Time | [RTL_LAND_DELAY](../advanced_config/parameter_reference.md#RTL_LAND_DELAY) | If return with loiter then land is selected you can also specify how long the vehicle will hold.
+
+> **Note** The return behavour is defined by [RTL_LAND_DELAY](../advanced_config/parameter_reference.md#RTL_LAND_DELAY). If negative the vehicle will land immediately. Additional information can be found in [Return mode](../flight_modes/return.md).
+
+
+### Land Mode Settings
+
+*Land at the current position* is a common [failsafe action](#failsafe_actions) that engages [Land Mode](../flight_modes/land.md). 
+This section shows how to set whether the vehicle will automatically disarm after landing. For Multicopters (only) you can additionally set the descent rate.
+
+![Safety - Land Mode Settings (QGC)](../../images/qgc/setup/safety_land_mode.png)
+
+The settings and underlying parameters are shown below:
+
+Setting | Parameter | Description
+--- | --- | ---
+Disarm After | [COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND) | Select checkbox to specify that the vehicle will disarm after landing, and enter delay after landing before disarming (must be non-zero). 
+Landing Descent Rate | [MPC_LAND_SPEED](../advanced_config/parameter_reference.md#MPC_LAND_SPEED) | Rate of descent (MC only).
+
+
+## Other Failsafe Settings {#failsafe_other}
+
+This section contains information about failsafe settings that cannot be configured through the *QGroundControl* [Safety Setup](#qgc_safety_setup) page.
+
+### Position (GPS) Loss Failsafe
+
+The *Position Loss Failsafe* is triggered if the quality of the PX4 position estimate falls below acceptable levels (this might be caused by GPS loss) while in a mode that requires an acceptable position estimate.
+
```

Comment:
@bkueng This is a bit of a guess based on my testing and @barzanisar testing in https://github.com/PX4/Firmware/issues/9336#issuecomment-386274299

It may not be correct because COM_POSCTL_NAVL is only listed in mission section, but it does appear to explain observed behaviour.

The main question is when the Fixed wing loiter happens (see line 162 below). It could happen immediately the failsafe is engaged, but before the action is triggered (ie you go to loiter then a manual mode after that) or it could be only in certain modes. Thoughts?

Reply:
I don't know about FW